__tests__/app.test.js

```diff
@@ -164,6 +164,54 @@ describe("GET /api/articles/:article_id", () => {
   });
 });
 
+describe("GET /api/articles/:article_id/comments", () => {
+  describe("Successful usage", () => {
+    test("Status 200: should respond with an array of comments objects, each of which with 'comment_id', 'votes', 'created_at', 'author' and 'body' properties corresponding to the passed in article id", () => {
+      const articleId = 1;
+      return request(app)
+        .get(`/api/articles/${articleId}/comments`)
+        .expect(200)
+        .then(({ body }) => {
+          expect(body).toBeInstanceOf(Array);
+          expect(body).toHaveLength(11);
+          body.forEach((user) => {
+            expect(typeof user).toBe("object");
+            expect(user !== null).toBe(true);
+            expect(!Array.isArray(user)).toBe(true);
```

Comment:
Not necessary to check if it's not an array when we already checked the type in line 178.

Reply:
Hi, I have used this test in all prior tests so far. I did always feel it was perhaps a little too thorough but it was my understanding that arrays and null will be classed as an "object" when doing typeof, is this not the case or is it still not worth checking for that it isn't an array?